### PR TITLE
Allow use of file:// in devfile.yaml for manifest and dockerfile

### PR DESCRIPTION
**Pull request by groeges**
**What type of PR is this?**
/kind cleanup

**What does does this PR do / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #?
Updates to allow use of file:// urls within devfile.yaml for alpha.deployment-manifest and alpha.build-dockerfile 

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -991,6 +991,54 @@ func DownloadFile(url string, filepath string) error {
...
+		// check to see if filepath exists
+		info, err := os.Stat(filepath)
+		if os.IsNotExist(err) || info.IsDir() {
+			return nil, errors.New(fmt.Sprintf("unable to read url: %s, %s", URL, err))
+		}
+
+		// read the bytes from the filepath
+		fileBytes, err = ioutil.ReadFile(filepath)
+		if err != nil {
+			return nil, errors.New(fmt.Sprintf("unable to read url: %s, %s", URL, err))
```
Comment (EnriqueL8) [inline pkg/util/util.go]: Would change this error message to unable to read file

File: pkg/util/util.go [hunk 2]
```diff
@@ -991,6 +991,54 @@ func DownloadFile(url string, filepath string) error {
...
+				return nil, errors.New("unable to determine current working directory")
+			}
+			// prepend the current working directory to the relatove filepath
+			filepath = fmt.Sprintf("%s/%s", cwd, filepath)
+		}
+
+		// check to see if filepath exists
+		info, err := os.Stat(filepath)
+		if os.IsNotExist(err) || info.IsDir() {
+			return nil, errors.New(fmt.Sprintf("unable to read url: %s, %s", URL, err))
```
Comment (EnriqueL8) [inline pkg/util/util.go]: Would change this to enable to find file

**Review by EnriqueL8 — COMMENTED** [review]
Looks good - too small comments on error messages

File: pkg/util/util.go (hunk 2 above)
Comment (groeges) [reply pkg/util/util.go]: Done

File: pkg/util/util.go (hunk 1 above)
Comment (groeges) [reply pkg/util/util.go]: Done

**Comment by groeges** [comment]
Also added some unit tests for util.LoadFileIntoMemory()

**Review by EnriqueL8 — APPROVED** [review]

**Pull request merged**